.github/workflows/pr-checker.yml

```
@@ -0,0 +1,33 @@
+# Created my @smog-root.
+on:
+  pull_request:
+    types: [opened, edited]
+
+jobs:
+  check_pr_description:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

actionlint

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)